__tests__/app.test.js

```diff
@@ -49,10 +49,39 @@ describe("GET categories", () => {
             .then(({
                 body
             }) => {
-                console.log(
-                    body.msg
-                )
                 expect(body.msg).toBe("Sorry we cant find that end point")
             })
     });
-})
\ No newline at end of file
+})
+
+describe('GET Reviews by id', () => {
+    test('return the correct review', () => {
+        return request(app)
+            .get("/api/reviews/1")
+            .expect(200)
+            .then(({
+                body
+            }) => {
+                expect(body.review.review_id).toBe(1)
+                expect(body.review.title).toBe('Agricola')
+                expect(body.review.designer).toBe('Uwe Rosenberg')
+                expect(body.review.owner).toBe('mallionaire')
+                expect(body.review.review_img_url).toBe('https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png')
+                expect(body.review.review_body).toBe('Farmyard fun!')
+                expect(body.review.category).toBe('euro game')
+                expect(body.review.votes).toBe(1)
+
+            })
+    });
+    test('when passed a number that doesnt exsist return 404', () => {
+        return request(app)
+            .get("/api/reviews/1000")
+            .expect(404)
+            .then((
+                body
```

Comment:
You've called the parameter in this arrow function `body` but be aware it is not invoked with the body!

it will be invoked with the `Response` object which has `.body` as a key!

You should either destructure `body` (as you did in the previous test) or rename this to `response` and access the `.body` key from it

```diff
.then(
-    (body) 
+    ({ body })
    => {})
```
> OR
```diff
.then(
-    (body) 
+    ( response )
    => {
+    const body = response.body
    })

```